packages/router/src/page-loader.tsx

```diff
@@ -2,12 +2,15 @@ import React, { useContext } from 'react'
 
 import isEqual from 'lodash.isequal'
 
+import { isPrerendering } from '@redwoodjs/web'
```

Comment:
I would _really_ love to keep these in the prerender package. I know you have an issue with the nodejs/ browser contexts, but check how I did the exports for `@redwoodjs/web/apollo` to mitigate that.

Reply:
I'll look through it again, but I mentioned in one of the reviews before, using webpack to do the import thing will mean it'll work for the end user, but not for us internally (right?).

Reply:
The current solution doesn't require webpack changes. It uses ordinary module resolution; check it out over here:
https://github.com/redwoodjs/redwood/blob/4c9ea69a2bdc534dae72905a5304b92c51a85db7/packages/web/apollo/package.json#L1-L4

Reply:
Hmmm, nice! I didn't see after you'd updated. 

Another issue trying to do it this way -> there's a circular dependency between router (isPrerendering) and prerender (LocationProvider). If you don't want it in web, perhaps another option is to keep it in internal?

Reply:
Hmpf. I'm not sure I'm following. Could you tell me which files to look at in order to understand this?

Reply:
I've pushed up the WIP code here. Changes: https://github.com/redwoodjs/redwood/commit/e7d238505a17644d34d9f6c241583a2a103fb13e

The problem is that 
a) Router depends on prerender
b) Prerender depends on Router

So what we could do is (but I don't like this idea)
1. Move browserUtils to the @redwoodjs/internal
2. Export browserUtils from prerender

Now router depends on internal, but not on prerender directly
And prerender still depends on router.